LIMS_IMAGE/web/templates/shopping/appendix_b.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html' %}

```

Comment:
for organization it makes more sense for both appendix b and shopping to be in orders.
Orders is the app that contains both. the templates are separated by the app they are related to

Reply:
there is already a folder in templates called 'orders' just move your files there, and adjust any references to them accordingly